doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1292,6 +1317,15 @@ Wi-Fi
 
 The issues in this section are related to the :ref:`ug_wifi` protocol.
 
+.. rst-class:: v3-0-0
+
+SHEL-3596: For nRF5340DK + nRF7002EK, XiP is not working after Wi-Fi interface down/up
```

Comment:
This issue has been fixed now. Please remove this from the known issues.

Reply:
Fix not yet backported. Keeping the issue.

Reply:
Waiting for @bama-nordic to confirm whether we have a fix.

Reply:
Thanks Bansi!